<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Views::ConsoleDockWidget</class>
 <widget class="QDockWidget" name="Core::Views::ConsoleDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>控制台</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="tool_bar_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="output_text">
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="../core.qrc">
     <normaloff>:/core/images/edit-delete-shred.png</normaloff>:/core/images/edit-delete-shred.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="action_warp_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../core.qrc">
     <normaloff>:/core/images/go-jump-locationbar.png</normaloff>:/core/images/go-jump-locationbar.png</iconset>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="action_scroll_to_end">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../core.qrc">
     <normaloff>:/core/images/scroll-to-end.png</normaloff>:/core/images/scroll-to-end.png</iconset>
   </property>
   <property name="text">
    <string>自动滚动</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../core.qrc"/>
 </resources>
 <connections/>
</ui>
